x/lockup/keeper/gas_test.go

```diff
@@ -13,8 +14,13 @@ var (
 )
 
 func (suite *KeeperTestSuite) measureLockGas(addr sdk.AccAddress, coins sdk.Coins, dur time.Duration) uint64 {
+	// fundAccount outside of gas measurement
+	err := simapp.FundAccount(suite.app.BankKeeper, suite.ctx, addr, coins)
+	suite.Require().NoError(err)
+	// start measuring gas
 	alreadySpent := suite.ctx.GasMeter().GasConsumed()
-	suite.LockTokens(addr, coins, dur)
+	_, err = suite.app.LockupKeeper.LockTokens(suite.ctx, addr, coins, dur)
+	suite.Require().NoError(err)
 	newSpent := suite.ctx.GasMeter().GasConsumed()
 	spentNow := newSpent - alreadySpent
 	return spentNow
```

Comment:
Good catch on this! 
One further thing we can do in the future is set a range and simply test if the avgGas and maxGas is within the range, or something similar to remove hard coding gas, considering that there could be further instances where we would have to manually change the hard-coded gas whenever there is a logic change within the method.

Reply:
hrmm, thats a good idea. (Basically lower the sensitivity to small noise that we care about)

Depends on our general attitude towards gas, where does tracking small changes matter. I tend to agree with you, at this stage, having small aberrations is totally fine, as long as we can bound the max, and that max and average don't get too far apart.